deploy/terraform-aws/cleanup-vpc.sh

```diff
@@ -55,9 +55,16 @@ get_pending_and_running_instance_ids_by_vpc_id() {
 instance_ids=$(get_pending_and_running_instance_ids_by_vpc_id "$VPC_ID")
 while [[ -n "$instance_ids" ]]
 do
-    echo "Terminating instances: $instance_ids"
-    aws ec2 terminate-instances --instance-ids "$instance_ids"
-    aws ec2 wait instance-terminated --instance-ids "$instance_ids"
+    for instance_id in $instance_ids
+    do
+        echo "Terminating instance: $instance_id"
+        aws ec2 terminate-instances --instance-ids "$instance_id"
+    done
+    for instance_id in $instance_ids
+    do
+        aws ec2 wait instance-terminated --instance-ids "$instance_id"
+        echo "Instance $instance_id terminated"
+    done
```

Comment:
I made these 2 loops because AWS recommend to limit the number of IDs passed to terminate-instances.

Reply:
nit: I understand what you mean however it looks like you could add `aws ec2 wait` to the first for loop. It looks like `aws ec2 wait` polls the aws describe instances api every 15 seconds until the desired state was reached. It would just mean that we wait on each iteration until state becomes terminated rather than sending the request for termination to every node then checking afterward if it completes or not.

Reply:
Yep we could make it a single loop. This would simplify things slightly, at the cost of more time during cluster shutdown.

I’m going to do what you suggest: given how brittle shell can be, better go for simple instead of trying to optimize for pennies of AWS credits.